### ISSUE TITLE
simple fix for Ubuntu 18.04 compatibility

### ISSUE THREAD

**Issue by ms2048** [SomajitDey/tunnel#6]
Changing the header from
`#!/usr/bin/env -S bash -O extglob`

to a two-liner
```
#!/usr/bin/env bash
shopt -s extglob
```

gives an additional bonus of the script fully working under Ubuntu 18.04.6 (according to my testing).

Otherwise, an option `-S` for `env` is unrecognized because Ubuntu 18.04 ships with `coreutils` 8.28. Support for the -S option was added to `env` in 8.30.

This should be forward-compatible. Please fix.

**Comment by SomajitDey** [comment]
I tried to incorporate your suggestion in https://github.com/SomajitDey/tunnel/commit/d680ecb74e69af7eee42f2a17720221febb10c83. But it resulted in the following error when running `tunnel` client:

```bash
bash: main: line 57: syntax error near unexpected token `('
bash: main: line 57: ` +([[:digit:]]) | ipfs)'
bash: error importing function definition for `main'
```

Hence had to discard this approach.

**Comment by ms2048** [comment]
in the commit https://github.com/SomajitDey/tunnel/commit/d680ecb74e69af7eee42f2a17720221febb10c83 you forgot to add the second line of the **two-liner**. It is important to do so, as I've explained.

**Comment by SomajitDey** [comment]
It is there at line 15: https://github.com/SomajitDey/tunnel/blob/d680ecb74e69af7eee42f2a17720221febb10c83/tunnel#L15.

**Comment by ms2048** [comment]
Ok, I only tested this as line 2.

It was also a long time ago.

Maybe some other changes made this incompatible just now?

My change worked well enough on both Ubuntu 18.04 and something newer back then... now I cannot re-test this reliably (time constrains, other commitments, etc.).

So - in case of this not working for you - I have to give up.

**Comment by SomajitDey** [comment]
Dear @ms2048, 

Finally managed to diagnose why `tunnel` client was throwing errors after using `shopt -s extglob`. Using `socat` SYSTEM: instead of EXEC: has resolved the issue in [v0.2.1](https://github.com/SomajitDey/tunnel/releases/tag/v0.2.1). Thanks for your original comment.
